pkg/madmin/keyvalue.go

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
We can call this `config-keyvalue.go` to be clearer.

Reply:
This is changing and moving to a `pkg/config` file